spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/README.adoc

```diff
@@ -1,141 +0,0 @@
-= Spring Cloud GCP Config Client Starter
```

Comment:
This file can't simply disappear.
We should move it to `spring-cloud-gcp-config`.